cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -516,7 +502,7 @@ features:
           string: String
         title: Change Data Type
```

Comment:
this is a button?

Reply:
I don't think menu items are counted as buttons, so we should change these too:
![image](https://user-images.githubusercontent.com/3895099/42661039-d2d4fe6c-85e1-11e8-98c1-97a294cbc3b3.png)